patches/core/ungoogled-chromium/fix-building-without-safebrowsing.patch

```diff
@@ -1,8 +1,18 @@
 # Additional changes to Inox's fix-building-without-safebrowsing.patch
 
+--- a/chrome/browser/browser_resources.grd
++++ b/chrome/browser/browser_resources.grd
+@@ -597,7 +597,6 @@
+       </if>
+       <if expr="not is_android">
+         <include name="IDR_TAB_RANKER_EXAMPLE_PREPROCESSOR_CONFIG_PB" file="resource_coordinator\tab_ranker\example_preprocessor_config.pb" type="BINDATA" />
+-        <include name="IDR_TAB_RANKER_PAIRWISE_EXAMPLE_PREPROCESSOR_CONFIG_PB" file="resource_coordinator\tab_ranker\pairwise_preprocessor_config.pb" type="BINDATA" />
```

Comment:
We should add an exception for this file to `devutils/update_lists.py` next to `example_preprocessor_config.pb`